README.md

```diff
@@ -1,4 +1,7 @@
 # Sierra Emulator
 
-This program takes a Cairo program in Sierra representation as input and "emulates" it to obtain the sierra state at each step. 
-It is meant as a debugging tool to compare the state against the results of running a Sierra program with Cairo Native. 
+This program takes a Cairo program in Sierra representation as input and "emulates" it to obtain the sierra state at each step.
+It is meant as a debugging tool to compare the state against the results of running a Sierra program with Cairo Native.
+
+## Running the Program
+`cargo run <SIERRA PROGRAM> <ENTRYPOINT ID>`
```

Comment:
The binary's arguments could be interpreted as destined for cargo using the current syntax. This can be fixed by forcing them to go to the binary:
```suggestion
`cargo run -- <SIERRA PROGRAM> <ENTRYPOINT ID>`
```